storeQuestionService/store-q-service.js

```diff
@@ -0,0 +1,102 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+const Question = require('./store-q-model');
+
+const app = express();
+const port = 8004;
+
+// Middleware to parse JSON in request body
+app.use(bodyParser.json());
```

Comment:
No hace falta añadir una dependencia nueva a body-parser pues express ya incluye dicha funcionalidad en sus últimas versiones. Se puede usar directamente así:
```suggestion
app.use(express.json());
```